README.md

```diff
@@ -1,10 +1,11 @@
-# YOURLS using Helm [![Listed in Awesome YOURLS!](https://img.shields.io/badge/Awesome-YOURLS-C5A3BE)](https://github.com/YOURLS/awesome-yourls/)
-
+# YOURLS using Helm 
 > The official [Helm](https://helm.sh) charts repository for [YOURLS](https://yourls.org).
 
 [![Build Status](https://github.com/YOURLS/charts/workflows/Charts%20CI/badge.svg)](https://github.com/YOURLS/charts/actions)
 [![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/yourls)](https://artifacthub.io/packages/search?repo=yourls)
 [![Chart Version](https://img.shields.io/badge/dynamic/yaml.svg?color=blue&label=chart&prefix=v&query=entries.yourls%5B0%5D.version&url=https%3A%2F%2Fcharts.yourls.org%2Findex.yaml)](https://artifacthub.io/packages/helm/yourls/yourls)
+[![Listed in Awesome YOURLS!](https://img.shields.io/badge/Awesome-YOURLS-C5A3BE)](https://github.com/YOURLS/awesome-yourls/)
+
```

Comment:
```suggestion
[![Listed in Awesome YOURLS](https://img.shields.io/badge/Awesome-YOURLS-C5A3BE)](https://github.com/YOURLS/awesome-yourls/)
```

Dropping the extra line, and removing the exclamation mark (let's keep the tone formal 😊)